src/commands/config-set.json

```diff
@@ -0,0 +1,28 @@
+{
+    "SET": {
+        "summary": "Set a configuration parameter to the given value",
```

Comment:
```suggestion
        "summary": "Set a configuration parameters to the given values",
```

Reply:
@yoav-steinberg i think we should first merge your doc PR and then we will re-generate the jsons so that config-set.json will include that fix

Reply:
@guybe7 Just merged with updated _commands.json_